File: app/src/main/java/com/ddancn/keepaccount/exception/TypeNameDuplicateException.java
package com.ddancn.keepaccount.exception;

/**
 * @author ddan.zhuang
 * @date 2019/10/16
 */
public class TypeNameDuplicateException extends Exception{

    public TypeNameDuplicateException() {
        super("类型名不要重复~");
    }
}

File: app/src/main/java/com/ddancn/keepaccount/exception/TypeNameDuplicateException.kt
package com.ddancn.keepaccount.exception

import com.blankj.utilcode.util.Utils
import com.ddancn.keepaccount.R

/**
 * @author ddan.zhuang
 * @date 2019/10/16
 */
class TypeNameDuplicateException
    : Exception(Utils.getApp().getString(R.string.setting_type_duplicate))